src/main/java/com/autilus/hw2706/Main.java

```diff
@@ -0,0 +1,27 @@
+package com.autilus.hw2706;
+
+import com.autilus.hw2706.controller.ConsoleHandler;
+import com.autilus.hw2706.dao.BetDaoImpl;
+import com.autilus.hw2706.dao.UserDaoImpl;
+import com.autilus.hw2706.lib.Injector;
+
+public class Main {
+
+    static {
+        try {
+            Injector.injectDependency();
+        } catch (IllegalAccessException e) {
+            e.printStackTrace();
```

Comment:
cs

Reply:
What wrong with my cs? IDEA told it's wright